Add feedback for the RHEL guide (#835)
SUSE/doc-susemanager: modules/client-configuration/pages/clients-rh.adoc

== Import Entitlements and CA Certificate

> {redhat} clients require a {redhat} certificate authority (CA) and entitlement certificate and entitlemente key.

Reviewer: ```suggestion
{redhat} clients require a {redhat} certificate authority (CA) and entitlement certificate and entitlement key.
```

It runs locally to track installed products and subscriptions.
Clients must be registered with the subscription manager to obtain certificates.

> Depending on where the {redhat} system is registered, the URL’s to which the system is authorized to replicate differ.

Reviewer: ```suggestion
{redhat} clients use a URL to replicate repositories.
The URL will change depending on where the {redhat} client is registered.
```

> {redhat} systems can be registered to three different infrastructures:

Reviewer: ```suggestion
{redhat} clients can be registered in three different ways:
```

> 1. {redhat} CDN (redhat.com)

Reviewer: ```suggestion
* {redhat} content delivery network (CDN) at redhat.com
```

> 2. {redhat} Satellite Server

Reviewer: ```suggestion
* {redhat} Satellite Server
```

> 3. RHUI (update infrastructure in the cloud)

Reviewer: ```suggestion
* {redhat} update infrastructure (RHUI) in the cloud
```

> This guide focuses on a system registered to {redhat} CDN (redhat.com). Register at least one system there with an authorized subscription to obtain repository content.

Reviewer: ```suggestion
This guide covers clients registered to {redhat} CDN. 
You must have at least one system registered to the CDN, with an authorized subscription for repository content.
```

[IMPORTANT]
====

> Entitlement certificates for RHUI (cloud-based systems) only authorize you to get content there.

Reviewer: ```suggestion
Entitlement certificates for RHUI (cloud-based systems) only allow you to download content, not repository data.
```

> Satellite certificates for client systems require a Satellite server and subscription: such client certificates are UNSUPPORTED with {productname} server.

Reviewer: ```suggestion
Satellite certificates for client systems require a Satellite server and subscription.
Clients using Satellite certificates are not supported with {productname} Server.
```

> ====

Reviewer: I feel like this admonition should be split into two: one about RHUI, one about Sat. Of course, then it means we have three admonitions in a row, which probably means it needs a little bit of re-thinking.

Author: @dvosburg ?

Reviewer: I'm happy to edit this later on, anyway, so consider it non-blocking.

[WARNING]
====
Entitlement certificates are embedded with expiration dates, which match the length of the support subscription.
To avoid disruption, you will need to repeat this process at the end of every support subscription period.
====

> {redhat} supplies the subscription-manager tool to manage subscription assignments. It runs locally on the client system to track installed products and subscriptions. Register to redhat.com with subscription-manager, then follow this procedure to obtain certificates.

Reviewer: ```suggestion
{redhat} supplies the subscription-manager tool to manage subscription assignments. 
It runs locally on the client system to track installed products and subscriptions. 
Register to redhat.com with subscription-manager, then follow this procedure to obtain certificates.
```
